modules/ROOT/partials/fragment-installing-operator-using-cli.adoc

```diff
@@ -0,0 +1,60 @@
+[start={next-step}]
+. Specify the namespace in which you want to install the Operator. You can create a new namespace or switch to an existing one.
+
+.. Create a new namespace:
++
+[source,bash,options="nowrap",subs="+quotes"]
+----
+$ kubectl create namespace  __<namespace-name>__
```

Comment:
Even if you create a new namespace using `kubectl` you still need to switch to it,
as demonstrated below.